action.yml

```diff
@@ -68,4 +68,4 @@ runs:
       if: ${{ inputs.url == '' || inputs.username == '' || inputs.password == '' }}
       shell: bash
       run: |
-        echo "osc is running without authentication. Either configure configuration in the 'setup-osc' action, or authenticate manually using 'osc auth login'."
+        echo "::warning::osc is running without authentication. Either configure authentication in the 'setup-osc' action (url, username and password), or authenticate manually using 'osc auth login'."
```

Comment:
What syntax is `::warning::`, Markdown? Would it make sense to add a reference to the commit message? Should there by a newline after it?

Reply:
Maybe use https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-warning-message with the right anchor?

Reply:
Done.
This is how it looks in the GitHub action summary:
![image](https://github.com/user-attachments/assets/842d495d-4945-4978-b69f-4da259f9d17a)

Reply:
> Should there by a newline after it?

I don't see a reason why it should be required.